src/pages/constitution/Constitution.tsx

```diff
@@ -59,8 +81,22 @@ class Constitution extends React.Component<Props> {
         </IonHeader>
         <IonContent>
           <div className="ion-padding">
-            <div className="akoma-ntoso" ref={this.rootRef}></div>
+            <div className="akoma-ntoso" ref={this.rootRef}>
+              <Portal id={this.props.match.params.id} constitution={this.constitution}>
```

Comment:
This doesn't work quite as expected, for two reasons:

1. it's changing the constitution document, which is used globally, rather than the rendered version. 
2. we need to be able to do this for ALL sections in a rendered akn document

I'm pushing a pull request to demonstrate what I'd like.